dclab/rtdc_dataset/plugins/plugin_feature.py

```diff
@@ -45,8 +55,15 @@ def find_plugin_feature_script(plugin_path):
 def remove_plugin_feature(plugin_instance):
     """Convenience function for removing a plugin instance"""
     if isinstance(plugin_instance, PlugInFeature):
+        if plugin_instance.feature_name in PlugInFeature.feature_names:
```

Comment:
Writing here for clarity: 
This will not always be `True` because the `PlugInFeature` setup happens in this order:
1. `PlugInFeature` class is initialised
2. The `feature_label`, `feature_name` etc are checked. If they fail, then you still have a `PlugInFeature` instance.

Therefore, the `PlugInFeature` instance can exist without a corresponding `feature_label` and `feature_name`.

If you try to delete these attributes without this check, it will most of the time just throw an error. However, if the user sets their `feature_name="deform"` and then `remove_plugin_feature` is run, `"deform"` will be removed from the `dclab.dfn` file!